### PR TITLE
New Reagents

### PR DESCRIPTION
**Pull request by Shadowmech88**
This PR adds two new reagents, the first to be exclusive to reactions between plant reagents.

Hemoscyanine, created by mixing solanine, hyoscyamine, and phenol. While present in a player's body, it removes 2u of blood each tick. 14 units will drain about 25% of someone's blood in total.

Anthracene, created by mixing zeaxanthin and radium. It can be injected in order to cause you to give off a dim glow, or can be splashed on things to cause them to glow for a short time, 1 second per unit splashed.

:cl:
 * rscadd: Added hemoscyanine, which gradually lowers one's blood level while in their system. It can be made by mixing solanine, hyoscyamine, and phenol.
 * rscadd: Added anthracene, which can be used to make people or things glow. It can be made by mixing zeaxanthin and radium.

### CODE REVIEW

**Comment by ghost** [comment]
I like it, especially the glowy one. But I'm not convinced letting players heal broken bones from nettle and ambrosia is balanced. In my opinion this chemical should be harder to get, or have a much harsher downside.

**Comment by FudgePucker** [comment]
What about requiring death nettles?

**Comment by Exxion** [comment]
That is a pretty harsh downside, considering you naturally heal brute but not burn
And that's a lot of burn

**Comment by FudgePucker** [comment]
Also that too.
Maybe also have it react with burn healing chemicals, removing both, and causing burn damage, meaning you can't just take burn meds with it to sidestep the downside.

**Comment by ghost** [comment]
You  do naturally heal burn though
and that's just 20 of it

**Comment by FudgePucker** [comment]
20 plus brute damage healed converted to burn damage.

**Comment by Exxion** [comment]
You're right. I was thinking of a discussion we had a while ago about MAKING you not heal burn.
But it's AT LEAST 20 (and possibly much more) PER BONE
If you have two or three broken bones, you're now in orange health at best, and possibly in crit

**Comment by ghost** [comment]
If you have 2-3 broken bones you already bled out and died

**Comment by Kurfursten** [comment]
Burn damage has weird properties. Small amounts of it (I think up to 10?) will heal over time, but if you have larger amounts it won't heal at all. For instance, if you get hit with a welder (15 burn) that will never heal unless you take medicine or salve it.

**Comment by ghost** [comment]
Brainstorming potential downsides here

First make it be like charcoal and clear itself from metabolism when healing a bone, so there can be a downside that is dangerous for just one broken bone but doesn't instantly murderize you with 2+ (you can do it at your own pace)

Idea one: Have it cause external bleeding, since you're probably already low on blood you really need to bandage it ASAP. Extra points: Have it knock you down in agony for 5 sec, so you want someone else three with you to help you bandage.

Idea 2: Have it cause toxin damage, bone healing only activates at >30 toxins, careful not to crit yourself bitch

Idea III: Seems like a good opportunity to have something neat added to the prehistoric fossil plants that currently have nothing going for them

**Comment by FudgePucker** [comment]
It is not a combat drug its a specialty drug.
If someone is REALLY fucked up it would result in death from those 20+ burn damages adding up to above 100.

I still think that making it so you can't have chemicals that treat burn damage (ALL OF THEM) working while you treat broken bones would prevent it from being used as a combat drug, as long as it can't be mixed into a cocktail of healing drugs it won't destroy balance.

I like the idea of hydrophonics being able to supplement medbay like this. Its more time consuming to get chemicals compared to just pushing some buttons, but it offers more.

**Comment by Kurfursten** [comment]
>Hydro**phonics**

https://www.youtube.com/watch?v=Kuw8YjSbKd4

**Comment by RFultra** [comment]
u:b:date wiki

**Comment by sonime** [comment]
updade wigi :DDDDDDDDDDDD

**Comment by clusterfack** [comment]
What the hell, why are you adding a drug that almost completely replaces surgery with cryo?

**Comment by Exxion** [comment]
Actually, that's a good point. Its drawback is pretty irrelevant inside the magic healing tube.

**Comment by Shadowmech88** [comment]
Are there any drawbacks that *wouldn't* be irrelevant inside the magic healing tube?

**Comment by clusterfack** [comment]
No, cryo drugs right now have the ability to heal every other kind of damage. Broken bones were one of the things it COULDN'T heal.

**Comment by Ait1** [comment]
https://www.youtube.com/watch?v=bjfFRwl5RM0

**Comment by Exxion** [comment]
I mean
You could just make it not work at low body temperatures

**Comment by Shadowmech88** [comment]
Yeah but then they could just fix their bones and *then* stick them into the healing tube to heal all the burn damage away. It'd work as long as the burn damage doesn't outright kill them.

**Comment by Orangerush** [comment]
I am glad that our proud coders are working towards making medbay obsolete

**Comment by Exxion** [comment]
Yeah, but that would at least be marginally better than it working *in* the tube

Cryo was a mistake

**Comment by ghost** [comment]
I suggested external bleeding earlier, that wouldn't play nice with Cryo.
I mean you wouldn't bleed inside Cryo but the doctor would have to remember to bandage you as soon as you get ejected and stuff.

**Comment by clusterfack** [comment]
In that case people could easily combine with bicardine overdoses

**Comment by ghost** [comment]
External bleeding and internal bleeding are different things

**Comment by ghost** [comment]
Unacceptable.  The only reasonable solution to bone fixing juice being added is to add bone hurting juice.  I say this as someone who cares about the natural balance of not only our world, but all adjacent.

**Comment by Exxion** [comment]
1. That meme was never funny in the first place, and it's been months
2. There are *at least* three chems that deal brute and thus can break bones

**Comment by ghost** [comment]
All jokes aside.  Osteopyrum seems a bit absurd.  Throwing a few plants together to "cure" broken bones with a pill isn't the type of medbay direction that I think we should head.

**Comment by Shadowmech88** [comment]
General consensus seems to be that osteopyrum isn't acceptable in its current state.

However, I'd like to know how many people would be fine with it if it had heavier downsides, like external bleeding/not working in cold temps, and how many people simply dislike the idea of healing broken bones via chems as a whole.

**Comment by ghost** [comment]
This may be a bit too wacky, but I think an interesting and acceptable compromise would be for it to only work in hot temperatures along with the current setbacks.  A pyrogenics drug if you will.  It would certainly be interesting.

**Comment by Shadowmech88** [comment]
That is interesting. Would it see use if it required a heat chamber, though? It seems like at that point even surgery would be faster.

**Comment by Kurfursten** [comment]
If the threshold was low, people would use hot showers

**Comment by FudgePucker** [comment]
I still like my idea of it not being able to mix with burn healing chemicals (all of them).
Can't treat the damage until it leaves the system.
Stick the chemical into cyro and the entire mix is mostly fucked.
Maybe have it cause burn damage just by being in the body.

**Comment by Kuranon** [comment]
I don't like the idea of a chemical healing broken bones at all

**Comment by Intigracy** [comment]
So what's up with this?

**Comment by Shadowmech88** [comment]
This isn't ready to be merged until osteopyrum is changed or removed.
I was waiting for more responses to decide which of those options to take.

**Comment by Intigracy** [comment]
Just remove it until you have a better idea on how to do it.

**Comment by Shadowmech88** [comment]
I've removed osteopyrum, for now.

**Comment by Shadowmech88** [comment]
Also I just remembered that uranium/radium can be obtained via Botany, from glowshrooms/glowberries.
Should I alter the anthracene recipe to require one of those instead of mescaline?

**Comment by Intigracy** [comment]
Shrug

Conflicts though

**Comment by SonixApache** [comment]
@Shadowmech88 fix conflicts

**Comment by Shadowmech88** [comment]
No one's answered whether they think I should change the anthracene recipe yet.

**Comment by Intigracy** [comment]
How hard are zeaxanthin and mescaline to make?

**Comment by Exxion** [comment]
Read the OP again

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -5782,3 +5782,86 @@ var/global/list/tonio_doesnt_remove=list("tonio", "blood")
 	id = THYMOL
 	description = "Thymol is used in the treatment of respiratory problems."
 	color = "#790D27" //rgb: 121, 13, 39
+
+//End of plant-specific reagents
```
Comment (SonixApache) [inline code/modules/reagents/Chemistry-Reagents.dm:5794]: why are there plant-specific combo reagents below this comment?

**Comment by SonixApache** [comment]
So it's created by mixing plant-imidazoline and plant-spacedrugs? Seems fine to me.

File: code/modules/reagents/Chemistry-Reagents.dm (hunk 1 above)
Comment (Shadowmech88) [reply code/modules/reagents/Chemistry-Reagents.dm:5794]: I intended that section to be for reagents that can be found in plants, not reagents obtained by mixing reagents found in plants.

**Comment by Shadowmech88** [comment]
Yeah, but I didn't remember when I made the recipe that glowberries/glowshrooms were a thing.
Would it make more sense for one of those to be used in making a chem whose sole purpose is glowing?

**Comment by SonixApache** [comment]
Probably. I'm not sure how you set it up, but most logical would be tying it to the bio-luminescence gene.

**Comment by Shadowmech88** [comment]
That'd necessitate it just being a chemical found in bioluminescent plants as opposed to the product of a chemical reaction, wouldn't it?
I'm not sure I know enough about plant code to go about having a gene add a reagent to plants.

**Comment by SonixApache** [comment]
I was thinking more in the lines of a chemical (that's tied to bio-luminosity? Found only in glowshrooms/berries?) that you **mix** with  zeaxanthin and mescaline to make anthracene.
The chemical doesn't have to do anything necessarily, it could just be a copy of phosphorus.

**Comment by Shadowmech88** [comment]
Well, radium/uranium is already found in glowshrooms and glowberries, I was planning on just putting one of those in the recipe.

**Comment by Intigracy** [comment]
You could just add the chem to the plant(s) instead of using chems in the plant to make the chem.

**Comment by SonixApache** [comment]
uranium +  zeaxanthin + mescaline

How dim is this glow?

**Comment by Shadowmech88** [comment]
It's this bright.
image

**Comment by Shadowmech88** [comment]
* Anthracene now requires only zeaxanthin and radium to create. These can be obtained from carrots and glowshrooms, respectively.

**Comment by ghost** [comment]
Holler when, update the wiki, and we'll merge it

**Comment by Shadowmech88** [comment]
@Skullyton 
Wiki is updated, this is ready to go.

**Comment by ghost** [comment]
In the future, notify people via the discord.

The notification via github kinda gets swept away as there's no difference between it and just random chatter in issues and PRs

**Pull request merged**